Classes/Controller/RoutingController.php

```diff
@@ -326,6 +326,23 @@ protected function initTSFE()
         //\TYPO3\CMS\Frontend\Page\PageGenerator::pagegenInit();
     }
 
+    /**
+     * Determine from extension configuration, if HTTP debug header can be added.
+     *
+     * @return boolean
+     */
+    public function supressHttpDebugHeader()
+    {
+        if (isset($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['routing'])) {
+            $extensionConfiguration = unserialize($GLOBALS['TYPO3_CONF_VARS']['EXT']['extConf']['routing']);
+            if (is_array($extensionConfiguration) && array_key_exists('supressHttpDebugHeader', $extensionConfiguration)) {
```

Comment:
If I'm not misleading, the check for `is_array`, `array_key_exists` and the next one for `1` could be simplified as:

```
if (!empty($extensionConfiguration['suppressHttpDebugHeader']) && $extensionConfiguration['suppressHttpDebugHeader'] === '1') {
```